src/test/java/ReadingRangeCountTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Test;
+import java.util.*;
+import static org.junit.Assert.*;
+
+public class ReadingRangeCountTest {
+    ReadingRangeCount readingRangeCount = new ReadingRangeCount();
+
+    @Test
+    public void countGivenRangeTest_invalid_startRange() throws Exception {
+    	List <Integer> inputRange = new ArrayList<>(Arrays.asList(4,5));
```

Comment:
Good start! 
Test cases need have variations in test data for eg. {3} {} {3,4,5,5,7,9,10} - include repeated numbers, single entry list etc.